src/modules/FioAddress/util.ts

```diff
@@ -223,17 +223,16 @@ export const refreshConnectedWalletsForFioAddress = async (
   const connectedWallets: StringMap = {}
   const connectedWalletsFromDisklet = await getConnectedWalletsForFioAddress(fioWallet, fioAddress)
   for (const wallet of wallets) {
-    const enabledTokens = await wallet.getEnabledTokens()
-    if (!enabledTokens.find((enabledToken: string) => enabledToken === wallet.currencyInfo.currencyCode)) {
-      enabledTokens.push(wallet.currencyInfo.currencyCode)
-    }
+    const { currencyConfig, enabledTokenIds } = wallet
+    const enabledCodes = enabledTokenIds.map(tokenId => currencyConfig.allTokens[tokenId].currencyCode)
+    enabledCodes.push(wallet.currencyInfo.currencyCode)
```

Comment:
I make this note because I notice the same pattern in a prior commit and figured this is required to avoid duplication.

Reply:
It seems like there was some confusion over whether `enabledTokens` included the parent currency or not. In the new `tokenId` era, the parent currency is always separate. Since we are using tokenId's now, the check is no longer needed.

Duplicated currency codes are still possible, even among tokens. If we really care about removing duplicates, then we would need to use a `Set` or something like that. The existing check would be inadequate in any case.